video/out/meson.build

```diff
@@ -8,7 +8,7 @@ protocols = [[wl_protocol_dir, 'stable/presentation-time/presentation-time.xml']
              [wl_protocol_dir, 'staging/content-type/content-type-v1.xml'],
              [wl_protocol_dir, 'staging/fractional-scale/fractional-scale-v1.xml'],
              [wl_protocol_dir, 'staging/single-pixel-buffer/single-pixel-buffer-v1.xml'],
-             ['protocols', 'xx-color-management-v4.xml']]
+             ['protocols', 'color-management-v1.xml']]
```

Comment:
Wouldn’t it be better to wait for wayland-protocols to get a release, and use it from `wl_protocol_dir`, avoiding the need to carry it in the repository?

Reply:
sure, could do that in this MR already